.gitignore

```diff
@@ -12,24 +12,27 @@ coverage
 dist
 /.package.json
 package-lock.json
-/packages/babel-runtime/core-js/**/*.js
-!/packages/babel-runtime/core-js/map.js
+
 /packages/babel-runtime/helpers/*.js
 !/packages/babel-runtime/helpers/toArray.js
 !/packages/babel-runtime/helpers/iterableToArray.js
 !/packages/babel-runtime/helpers/temporalRef.js
-/packages/babel-runtime/helpers/builtin/*.js
-!/packages/babel-runtime/helpers/builtin/toArray.js
-!/packages/babel-runtime/helpers/builtin/iterableToArray.js
-!/packages/babel-runtime/helpers/builtin/temporalRef.js
-/packages/babel-runtime/helpers/builtin/es6/*.js
-!/packages/babel-runtime/helpers/builtin/es6/toArray.js
-!/packages/babel-runtime/helpers/builtin/es6/iterableToArray.js
-!/packages/babel-runtime/helpers/builtin/es6/temporalRef.js
 /packages/babel-runtime/helpers/es6/*.js
 !/packages/babel-runtime/helpers/es6/toArray.js
 !/packages/babel-runtime/helpers/es6/iterableToArray.js
 !/packages/babel-runtime/helpers/es6/temporalRef.js
+
+/packages/babel-runtime-corejs2/helpers/*.js
+!/packages/babel-runtime-corejs2/helpers/toArray.js
+!/packages/babel-runtime-corejs2/helpers/iterableToArray.js
+!/packages/babel-runtime-corejs2/helpers/temporalRef.js
+/packages/babel-runtime-corejs2/helpers/es6/*.js
+!/packages/babel-runtime-corejs2/helpers/es6/toArray.js
+!/packages/babel-runtime-corejs2/helpers/es6/iterableToArray.js
+!/packages/babel-runtime-corejs2/helpers/es6/temporalRef.js
+/packages/babel-runtime-corejs2/core-js/**/*.js
+!/packages/babel-runtime-corejs2/core-js/map.js
```

Comment:
For anyone else reviewing, the reason why I added these `!` gitignore's earlier is so that we are able to view the changes in output for the runtime since we would change the build script and it had a bug before and we never viewed the diff. But maybe there is a better way to do this since it's hardcoded. Basically we want to catch issues with the script